src/objects/zcl_abapgit_object_fugr.clas.abap

```diff
@@ -136,7 +144,17 @@ ENDCLASS.
 
 
 
-CLASS ZCL_ABAPGIT_OBJECT_FUGR IMPLEMENTATION.
+CLASS zcl_abapgit_object_fugr IMPLEMENTATION.
+
+  METHOD constructor.
+
+    super->constructor(
+               is_item     = is_item
+               iv_language = iv_language ).
+
+    mv_corrnum = zcl_abapgit_default_transport=>get_instance( )->get( )-ordernum.
```

Comment:
wondering if this should be in the constructor or not, it means its also called for serialization

Reply:
Yeah, moved it.